src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-tabs/cephfs-tabs.component.html

```diff
@@ -6,8 +6,10 @@
     </cd-cephfs-detail>
   </tab>
   <tab i18n-heading
-       (selectTab)="softRefresh()"
-       heading="Clients: {{ clients.data.length }}">
+       (selectTab)="softRefresh()">
+    <ng-template tabHeading>
+      Clients <span class="badge badge-pill badge-tab">{{ clients.data.length }}</span>
```

Comment:
We are missing the translation:
```suggestion
      <ng-container i18n>Clients</ng-container><span class="badge badge-pill badge-tab">{{ clients.data.length }}</span>
```
We can also remove the `i18n-heading` in line 8

Reply:
Fixed.